be/src/agent/utils.cpp

```diff
@@ -16,32 +16,33 @@
 // under the License.
 
 #include "agent/utils.h"
+
 #include <arpa/inet.h>
-#include <cstdio>
 #include <errno.h>
-#include <fstream>
-#include <iostream>
 #include <netdb.h>
 #include <netinet/in.h>
-#include <sstream>
+#include <rapidjson/document.h>
+#include <rapidjson/rapidjson.h>
+#include <rapidjson/stringbuffer.h>
+#include <rapidjson/writer.h>
 #include <sys/socket.h>
 #include <sys/wait.h>
-#include <unistd.h>
-
-#include <boost/filesystem.hpp>
 #include <thrift/Thrift.h>
 #include <thrift/transport/TSocket.h>
 #include <thrift/transport/TTransportException.h>
 #include <thrift/transport/TTransportUtils.h>
-#include <rapidjson/document.h>
-#include <rapidjson/rapidjson.h>
-#include <rapidjson/stringbuffer.h>
-#include <rapidjson/writer.h>
+#include <unistd.h>
+
+#include <boost/filesystem.hpp>
+#include <cstdio>
```

Comment:
current Doris header including style is :
1. The corresponding header files of .cpp (if any)
2. C (standard) library;
3. C ++ (standard) library
4. other libraries
5. .h files of your own project.

And the files in each section is in alphabetical order.

Can you do it like this?

Reply:
Clang-format supports grouping includes via `IncludeCategories`. I think we can inherit [impala's style](https://github.com/apache/impala/blob/master/.clang-format), which is

1. The corresponding header files of .cpp (if any)
2. <*.h> which covers most C library
3. <*> which covers most C++ standard library
4. <*/*> which covers third-party libraries
5. "*" which covers doris's own header